### DISCUSSION TITLE
Welcome to chat-llama-discord-bot Discussions!

### DISCUSSION THREAD

**Discussion by mrcm8** [mrcm8/chat-llama-discord-bot#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ConflyctX** [comment]
Good job on the bot so far!

I had a question, and although it's more of a Automatic1111 related one. I was wondering how I would change the default hr_upscaler in the payload. I've tried numerical values, named, etc.. I looked around in the scripts of Auto but so far haven't found anything yet.

payload = { "prompt": image_prompt, "width": 640, "height": 512, "steps": 40, "restore_faces": True, "cfg_scale":7, "enable_hr":True, "denoising_strength":0.55, "hr_scale":2, "hr_second_pass_steps":20 } 

These are my current values, and I'd like to add a downloaded upscaler (4x-ultrasharp) to it because I'm not a big fan of the default Latent one. If you have any clue, please.. fill me in. 

As for idea, maybe a command similar to the "Take a picture" one, would be something like "Remind me in" + time prompt + objective. When the time passes, make the LLM do an output based on the prompt that you added to remind you for if the bot is still active. 

Cheers,

**Reply by mrcm8** [reply]
Added a way to control the payload in the character yaml file now, including the upscaler

The reminder idea is cool, but would be coolest if you could use the LLM to interpret the message... which would be complicated. Gonna mull on it. Might chicken out and just make a /remindme command

**Comment by mrcm8** [comment]
It defaults back to the latent scaler even though you've set A111 to use 4x-ultrasharp? Shame. I remember seeing some extra postprocessing  stuff in the API, it might be there. I'll have a little look around. It's all very barebones at the moment as you may have noticed.

I like your reminder / notification idea! Would be even better if it could be interpreted naturally though.. The commands feel so brute force. Maybe I could hijack the regular prompt whenever someone says "remind me" and then have it access some function..

**Comment by ConflyctX** [comment]
Good job on the recent updates, love the one where the AI talks about the image it has generated!
